play-json/src/main/resources/reference.conf

```diff
@@ -0,0 +1,20 @@
+lagom.serialization.play-json {
+
+  # This needs to be configured by the application and pointed towards an
+  # object (singleton) implementing com.lightbend.lagom.scaladsl.playjson.PlayJsonSerializationRegistry
+  # TODO set it to a default place, like global in play?
+  # serialization-registry = "com.myapp.SerializationRegistry
```

Comment:
I guess we can have it be a singleton/module just as well if someone likes that better.